lsp-ui-doc.el

```diff
@@ -98,6 +98,12 @@ This affects the position of the documentation when
                  (const :tag "At point" at-point))
   :group 'lsp-ui-doc)
 
+(defcustom lsp-ui-doc-side 'right
+  "Which side to display the doc."
+  :type '(choice (const :tag "Left" left)
+                 (const :tag "Right" right))
+  :group 'lsp-ui-doc)
```

Comment:
Can you document this variable in our README? Thanks!

Reply:
Done.